Assets/MixedRealityToolkit/Utilities/CameraCache.cs

```diff
@@ -20,7 +20,15 @@ public static Camera Main
         {
             get
             {
-                var mainCamera = cachedCamera == null ? Refresh(Camera.main) : cachedCamera;
+                var mainCamera = Camera.main;
+
+                if (mainCamera == null)
```

Comment:
I'm not sure this most recent change makes sense in the flow of this method. Previously, it was written so it:
1. Checks for a main-tagged camera.
2. If it doesn't find one, search the scene for any non-tagged cameras.
3. If it doesn't find any non-tagged cameras, create one.
4. If it does find a non-tagged camera, use that.
5. If it finds more than one, look for name or use the first one.
6. Return

Now, searching for a non-tagged camera will never be hit (because we always create a new main-tagged camera if one doesn't exist), and we may create a new camera even if one already exists:

1. Checks for a main-tagged camera.
2. If it doesn't find one, create a main-tagged camera.
3. Return

Reply:
I considered this as well but I actually think this is an improvement.

There may be cases where you have camera(s) in your scene(s) that aren't meant to serve as a main camera. In those cases by leaving them untagged you can be confident they won't be usurped by the MRTK.

Reply:
Ahh, good call, yeah. Should we remove much of the rest of this method then, to help prevent confusion, etc?

Reply:
Yeah we can simplify it. And we can re-add a check for the cached camera since we lose the benefit of avoiding the tag check otherwise.